docs/api-reference/core/luma.md

```diff
@@ -151,7 +151,7 @@ Note that while you cannot directly attach a luma.gl `Device` to a WebGL 1 `WebG
 ### `luma.registerAdapters()`
 
 ```typescript
-luma.registerAdapters(adapters?: (typeof Device)[]): void;
+luma.registerAdapters(adapters?: Adapter[]): void;
```

Comment:
What is the difference between a Device class and an Adapter?

Reply:
- A Device is a wrapper around a WebGL context or GPUDevice that lets you create resources and query limits and features. 
- An adapter is a factory that lets you create Device's for a specific backend API.